public/app/plugins/datasource/loki/configuration/QuerySettings.tsx

```diff
@@ -27,64 +25,64 @@ export const QuerySettings = (props: Props) => {
         />
       }
     >
-      <div className="gf-form-inline">
-        <div className="gf-form">
-          <FormField
-            label="Maximum lines"
-            labelWidth={11}
-            inputWidth={20}
-            inputEl={
-              <input
-                type="number"
-                className="gf-form-input width-8 gf-form-input--has-help-icon"
-                value={maxLines}
-                onChange={(event) => onMaxLinedChange(event.currentTarget.value)}
-                spellCheck={false}
-                placeholder="1000"
-              />
-            }
+      <InlineField
+        label="Maximum lines"
+        htmlFor="loki_config_maxLines"
+        labelWidth={22}
+        tooltip={
+          <>
+            Loki queries must contain a limit of the maximum number of lines returned (default: 1000). Increase this
+            limit to have a bigger result set for ad-hoc analysis. Decrease this limit if your browser becomes sluggish
+            when displaying the log results.
+          </>
+        }
+      >
+        <Input
+          type="number"
+          id="loki_config_maxLines"
+          value={maxLines}
+          onChange={(event: { currentTarget: { value: string } }) => onMaxLinedChange(event.currentTarget.value)}
```

Comment:
Please use a proper type for the Event, there's no need to define a custom one.

Reply:
Changed